server/openapi_server/models/DB_utils.py

```
@@ -0,0 +1,601 @@
+        return document
+    except Exception as e:
+        raise RuntimeError(f"Error finding a document in MongoDB collection '{collection_name}': {e}")
+
+
+#############################################################################################################################
+
+# Initialize password hashing context
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+
+def validate_object_id(user_id: str) -> ObjectId:
```

Ruff (0.11.9)

Redefinition of unused `validate_object_id` from line 2

(F811)